src/transformState.js

```diff
@@ -46,7 +46,22 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
-}
+  for (const keys of transforms) {
+    if (keys.operation === 'addProperties') {
+      Object.assign(state, keys.properties);
+    } ;
+
+    if (keys.operation === 'removeProperties') {
+      for (const key of keys.properties) {
+        delete state[key];
+      };
+    };
 
+    if (keys.operation === 'clear') {
```

Comment:
строки прийнято виносити в окрему змінну константу, для подальшого використання в кількох місцях.
 ```
const OPERATIONS = {
  add: 'addProperties',
  remove: 'removeProperties',
  clear: 'clear'
}
```
потім ти можеш використовувати це ось так 
```suggestion
    if (transforms[i].operation === OPERATIONS.add) {
```
І якщо ці операції будуть використовувати в 2-3 місцях в програмі і в разі якщо треба буде змінити строку на щось інше, тобі не треба буде шукати і робити зміни по всіх файлах, достатньо змінити тільки строку в константі.